framework/Models/EnsembleModel.py

```diff
@@ -222,14 +221,12 @@ def initialize(self,runInfo,inputs,initDict=None):
       @ Out, None
     """
     # store the job ids for jobs that we haven't collected optional output from
-    self.tempOutputs['uncollectedJobIds'] = []
     # collect name of all the outputs in the Step
     outputsNames = []
     if initDict is not None:
       outputsNames = [output.name for output in initDict['Output']]
 
     # here we check if all the inputs inputted in the Step containing the EnsembleModel are actually used
-    # -> but why?  Does this check prevent problems later?
```

Comment:
Did this query get answered?

Reply:
As discussed during the meeting, the reason is just to avoid that unused inputs (and files) are copied and/or pickled if not used.